docs/reference/images/index.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Images provide an Operating System for your server.
+---
+
+# Images
+
+Public [Glance images](../../howto/openstack/glance/index.md) in {{brand}} contain regularly updated minimal versions of server operating systems.
+
+These images also contain the [cloud-init](https://cloudinit.readthedocs.io) package applicable to the operating system, to support the injection of SSH public keys and other and user data.
+
+## Naming conventions
+
+### Image names
+
+Image names in {{brand}} follow a convention, which can be summarized as `${NAME} ${VERSION_ID} ${CODENAME} ${ARCH}`:
+
+* `NAME`: Operating System name, such as `Ubuntu`, `Debian`, `Rocky`, etc.
+* `VERSION_ID`: Operating System version, as in `22.04`, `11`, `9`, etc.
+* `CODENAME`: Operating System codename, if present, for example: `Jammy Jellyfish`, `Bullseye`, etc.
```

Comment:
Yeah nah. :) We'll keep this to images we actually make available.

Reply:
Uhm, sorry, I meant to say "Blue Onyx" (Rocky 9, https://git.rockylinux.org/original/rpms/rocky-release/-/blob/r8/code).

Reply:
@noonedeadpunk Any thoughts on why we *don't* include the code name in our Rocky images?

Reply:
To be frank I didn't know it has a codename - thought it's in line with CentOS that didn't, and never saw rocky folks referring it by codename, so just inertia of thinking is the reason :(

Reply:
I also heard of no one, ever, mentioning Rocky by a codename. But then I thought there must be one, so I went hunting on Wikipedia (https://en.wikipedia.org/wiki/Rocky_Linux).

Reply:
@noonedeadpunk Okay, I guess it's fine then that we keep the "if present" bit but not mention any codename for Rocky, since our images names don't include it either.

Reply:
We can add codenames to rocky as well, if we think it's correct and worth doing